toolkit/tools/specreader/specreader.go

```diff
@@ -498,40 +522,21 @@ func parseProvides(rpmsDir, toolchainDir string, toolchainRPMs []string, srpmPat
 	return
 }
 
-// parsePackageVersions takes a package name and splits it into a set of PackageVer structures.
+// parsePackageVersions takes a package string and splits it into a set of PackageVer structures.
 // Normally a list of length 1 is returned, however parsePackageVersions is also responsible for
 // identifying if the package name is an "or" condition and returning all options.
-func parsePackageVersions(packagename string) (newpkgs []*pkgjson.PackageVer, err error) {
-	const (
-		NameField      = iota
-		ConditionField = iota
-		VersionField   = iota
-	)
+func parsePackageVersions(packageString string) (newPackages []*pkgjson.PackageVer, err error) {
+	// If the first character of the packageString is a "(" then it's an "or" condition.
+	if packageString[0] == '(' {
```

Comment:
I think `parsePackageVersions()` was the only questionable caller that may have passed bad input, but we might want to run a trim on the input if we are going to be checking the 1st character for `'('`.